src/neural/encoder.cc

```diff
@@ -80,6 +86,18 @@ InputPlanes EncodePositionForNN(const PositionHistory& history,
 
     const int repetitions = position.GetRepetitions();
     if (repetitions >= 1) result[base + 12].SetAll();
+
+    if (history_idx < 0 && !board.en_passant().empty()) {
+      const auto idx = GetLowestBit(board.en_passant().as_int());
```

Comment:
`// If en passant flag is set, undo last pawn move by removing the pawn from the new square and putting into pre-move square`